sr_error_reporter/src/UnderactuationErrorReporter.cpp

```diff
@@ -58,6 +53,11 @@ UnderactuationErrorReporter::UnderactuationErrorReporter(ros::NodeHandle& node_h
   {
     ROS_ERROR("UnderactuationErrorReporter did't find robot_description_semantic on parameter server");
   }
+
+  trajectory_subscriber_left_ = node_handle.subscribe("/lh_trajectory_controller/state", 1,
+    &UnderactuationErrorReporter::trajectory_callback_left, this, ros::TransportHints().tcpNoDelay());
+  trajectory_subscriber_right_ = node_handle.subscribe("/rh_trajectory_controller/state", 1,
```

Comment:
Shouldn't we avoid globally defining topics? In case we choose to namespace the hands further in the future.

Reply:
This story/PR doesn't really aim to refactor this node. These changes are only added to fix an edge case. If we wish to revisit this node/package at some point, we should also make sure we want to keep it in the first place

Reply:
sounds good